x-pack/metricbeat/module/azure/data.go

```diff
@@ -118,6 +123,20 @@ func managePropertyName(metric string) string {
 	return resultMetricName
 }
 
+// replaceUpperCase func will replace upper case with '_'
```

Comment:
Could you add some unit tests for `replaceUpperCase` function please?

Reply:
added some tests!

Reply:
Thanks!